UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -709,6 +702,7 @@ PlatformBootManagerBeforeConsole (
   //
   FilterAndProcess (&gEfiPciRootBridgeIoProtocolGuid, NULL, Connect);
 
+  PlatformConsoleInit ();

```

Comment:
Not really related to this change, but have you tested if output still works if console is initialized before `EfiBootManagerDispatchDeferredImages()`?

Reply:
PlatformConsoleInit simply adds a PCI UART, so it will work as soon as Root Bridge IO is connected

Reply:
Yes, I am sure: https://github.com/Dasharo/edk2/blob/dasharo/UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformConsole.c#L538

It handles only PS2 keybaord and Intel PCI Serial ports